src/client.rs

```diff
@@ -764,11 +770,7 @@ where
             server.claim(self.process_id, self.secret_key);
             self.connected_to_server = true;
 
-            // Update statistics.
-            if let Some(last_address_id) = self.last_address_id {
-                self.stats
-                    .client_disconnecting(self.process_id, last_address_id);
-            }
```

Comment:
`client_disconnecting` has no use for `last_address_id` (Based on my reading of the code)

Reply:
That's what I thought as well.

But it's used here, https://github.com/levkk/pgcat/blob/main/src/stats.rs#L398 to remove from the correct dict here, https://github.com/levkk/pgcat/blob/main/src/stats.rs#L456